kodi_voice/kodi.py

```diff
@@ -50,6 +50,14 @@ def sanitize_name(media_name, remove_between=False, normalize=True):
   name = name.strip()
   return name
 
+def sanitize_channel(name, lang='en'):
+  name = name.lower()
+  name = name.replace("+", " plus ")
+  name = re.sub(r"\sch\s", " channel ", name)
+  name = re.sub("^channel", "", name)
+  name = re.sub(r"(?<=\D)(?=\d)|(?<=\d)(?=\D)", " ", name)
+  name = words2digits(name, lang=lang)
```

Comment:
For text mangling, that's what the fuzzy matching is for.  It shouldn't do this in sanitize_channel().
If we need to make adjustments to get it to work more reliably, we can, but this isn't the place to do that.

Furthermore, in German, Amazon spits out number-words rather than digits.